src/pages/exclusive-content/components/copy-content.js

```diff
@@ -37,7 +37,7 @@ export function CopyContent(props) {
   return (
     <div>
       <Tooltip title={props.message}>
-        <Button tooltip onClick={() => copyContent(props.id)}>
```

Comment:
The Button component does not have this prop 🧐 and it was causing console errors